.github/imports/inital-content-for-visual-tests.sql

```diff
@@ -16,8 +16,8 @@ INSERT IGNORE INTO `rex_clang` VALUES
 (2,'en','english',2,0,0);
 
 REPLACE INTO `rex_config` VALUES
-('structure','history','true'),
-('structure','version','true');
+('core','history','true'),
+('core','structure_version','true');
```

Comment:
würde es auch `structure_history` nennen, damit es expliziter ist. Kann ja ggf. auch noch andere Histories geben mal.

Reply:
Ich würde `structure_version` eher in `versioning` oder `version_management` ändern. Diese jetzigen Präfixe mit Bezug auf die alten AddOns finde ich im Core eher suboptimal.

Reply:
oder vielleicht `article_history`? und `article_version`, oder sogar `article_work_version`, weil das ist ja der wesentliche Aspekt, dass eine Arbeitsversion parallel zur Liveversion entsteht.